internals/reminders_test.py

```diff
@@ -37,18 +37,39 @@ def __init__(self, status_code=200, text='{}'):
 
 
 def make_test_features():
-  feature_1 = core_models.Feature(
-      name='feature one', summary='sum', owner=['feature_owner@example.com'],
-      category=1, ot_milestone_desktop_start=100)
+  feature_1 = FeatureEntry(
```

Comment:
feature_1 -> feature_entry_1
similar below for the other features that were changed to FeatureEntry

Reply:
I agree with this, and I will do this in the future. The hope is to replace the old Feature kind entirely, so eventually they will phase out. However, they should be differentiated while they're still in the code together.

Reply:
Oh. I didn't know `Feature` is going away eventually. Thanks for that. I only mentioned this change because the class changed from `Feature` to `FeatureEntry`. But if FeatureEntry will eventually become the Feature, we can keep it.